CMakeLists.txt

```diff
@@ -752,10 +772,14 @@ if(GAMMARAY_BUILD_UI AND (Qt5Widgets_FOUND OR QT_QTGUI_FOUND))
     add_subdirectory(3rdparty/kuserfeedback)
   endif()
   add_subdirectory(ui)
-  add_subdirectory(inprocessui)
+  if (NOT GAMMARAY_CLIENT_ONLY_BUILD)
+    add_subdirectory(inprocessui)
+  endif()
   if(NOT GAMMARAY_PROBE_ONLY_BUILD)
     add_subdirectory(client)
-    add_subdirectory(app)
+    if (NOT GAMMARAY_CLIENT_ONLY_BUILD)
```

Comment:
Isn't "app" the macos client/launcher combination and thus should be build in client-only mode too?

Reply:
Fixed